File: Java/AlgorithmsPractice/src/findMissing/FindMissingNumberTest.java
package findMissing;

import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class FindMissingNumberTest {
    FindMissingNumber testObj1;

    FindMissingNumber testObj2;


    @BeforeEach
    void setUp() {
        this.testObj1 = new FindMissingNumber(new int[] {10, 8, 7, 6, 5, 4, 3, 2, 1});
        this.testObj2 = new FindMissingNumber(new int[] {});
    }

    @Test
    void getResult() {
        assertEquals(this.testObj1.getResult(), 9);
        assertEquals(this.testObj2.getResult(), 1);
    }
}

File: Kotlin/AlgorithmsPractice/test/findMissing/FindMissingNumberTest.kt
package findMissing

import org.junit.jupiter.api.Test

import org.junit.jupiter.api.Assertions.*

class FindMissingNumberTest {

    val findMissingNumber1 = FindMissingNumber(intArrayOf(1, 2, 4, 5, 6))

    val findMissingNumber2 = FindMissingNumber(intArrayOf())

    val findMissingNumber3 = FindMissingNumber(intArrayOf(5, 2, 4, 1, 6))


    @Test
    fun getResult() {
        assertEquals(3, findMissingNumber1.getResult())
        assertEquals(1, findMissingNumber2.getResult())
        assertEquals(3, findMissingNumber1.getResult())
    }
}